parsing/g_prim.mlg

```diff
@@ -66,9 +66,12 @@ GRAMMAR EXTEND Gram
   var: (* as identref, but interpret as a term identifier in ltac *)
     [ [ id = ident -> { CAst.make ~loc id } ] ]
   ;
-  identref:
+  lident:
     [ [ id = ident -> { CAst.make ~loc id } ] ]
   ;
+  identref:
+    [ [ id = lident -> { id } ] ]
+  ;
```

Comment:
Why not removed?

Reply:
If enough energy/time for the whole plan, the idea would be to have a deprecation phase of `identref` (and then `lident` too).